### ISSUE TITLE
Script to logout of all accounts signed in on the browser

### ISSUE THREAD

**Issue by hrishikeshcs** [hrishikeshcs/Scripts#2]
Create a script in any language to sign out of all accounts (on any website) when the browser is closed. It can perform something as straightforward as clearing the cookies.

**Comment by rexdivakar** [comment]
Are  u suggesting to clear all browser cookies ?

**Comment by hrishikeshcs** [comment]
@rexdivakar Yes. If there's any other way you'd like, then you can do it that way too.
